packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -65,13 +66,11 @@ export class ScmTreeWidget extends TreeWidget {
     ) {
         super(props, model, contextMenuRenderer);
         this.id = 'resource_widget';
-    }
-
-    @postConstruct()
-    protected init(): void {
-        super.init();
         this.addClass('groups-outer-container');
+    }
 
+    protected onActivateRequest(msg: Message): void {
```

Comment:
it does not look good, whenever widget is focused we add yet another listener

Reply:
You are right, it appears onActivateRequest is called every time the tab icon is selected.  I've moved to onAfterAttach which is a better place.